sdk/eventhub/azure-eventhub/azure/eventhub/_utils.py

```diff
@@ -232,7 +234,7 @@ def parse_sas_credential(credential: AzureSasCredential) -> Tuple[str, Optional[
 def transform_outbound_single_message(
         message: Union[AmqpAnnotatedMessage, EventData],
         message_type: Type[EventData],
-        to_outgoing_amqp_message: Callable
+        to_outgoing_amqp_message: Callable[[AmqpAnnotatedMessage], Union[uamqp_message, pyamqp_message]]
```

Comment:
ignore me b/c I am lacking some context for how we do things in this library, but just wondering why we have the lower snakecase names for types? This kind of reads like it returns a union of functions to me.

Reply:
ah my mistake will update